### DISCUSSION TITLE
🎁 Give Away 🎁

### DISCUSSION THREAD

**Discussion by cristiane13091991** [hashlips-lab/nft-erc721-collection#263] [Ideas]
🤔 How can I do to give away an NFT and this is not part of the MINTING because it would be reserved through the wallet by the winning person.

🙏 I wait your answer,
🚀 Thank you so much for all you do for the community. successes!

**Comment by GitBasil** [comment]
I was also about to ask the same question !!

**Reply by Shiba-Matic** [reply]
I was also about to ask the same question !!

**Comment by it-is-zods** [comment]
As @DestinyDevTeam mentioned, run the minting as normal and whenever you want to do a giveaway simply go to the contract and call mintForAddress - you would only need to pay for gas to send any given of tokens (_mintAmount) to a specific wallet (_receiver)
